railties/lib/rails/commands/runner/runner_command.rb

```diff
@@ -51,14 +51,24 @@ def perform(code_or_file = nil, *command_argv)
               eval(code_or_file, TOPLEVEL_BINDING, __FILE__, __LINE__)
             end
           rescue SyntaxError, NameError => e
```

Comment:
I haven't fully thought through how these two error classes could have been raised but would it have made more sense to split this into two rescue blocks? Im assuming `SyntaxError` would not be raised from this case.